angular-hometask/src/app/config/config/config.component.ts

```diff
@@ -0,0 +1,61 @@
+import { Component } from '@angular/core';
+import { ConfigService, Config } from '../config.service';
+
+@Component({
+  selector: 'app-config',
+  templateUrl: './config.component.html',
+  styleUrls: ['./config.component.scss'],
+  providers: [ConfigService]
+})
+export class ConfigComponent {
```

Comment:
What purpose of this component?

Reply:
There is no purpose, I made it as part of angular documentation and decided to leave in case I need it later. Will remove it.

Reply:
Done a4ca62512355de4288a134a4485a60f0a8230084